src/django/api/views.py

```diff
@@ -2084,6 +2098,38 @@ def dissociate(self, request, pk=None):
             facility, context=context).data
         return Response(facility_data)
 
+    @action(detail=True, methods=['POST'],
+            permission_classes=(IsRegisteredAndConfirmed,),
+            url_path='report')
+    @transaction.atomic
+    def report(self, request, pk=None):
+        """
+        Report that a facility has been closed or opened.
+
+        ## Sample Request Body
+
+            {
+                "closure_state": "CLOSED",
+                "reason_for_report": "This facility was closed."
+            }
+        """
+        try:
+            facility = Facility.objects.get(pk=pk)
+        except Facility.DoesNotExist:
+            raise NotFound('Facility with OAR ID {} not found'.format(pk))
+
+        contributor = request.user.contributor
+        facility_activity_report = FacilityActivityReport.objects.create(
+            facility=facility,
+            reported_by_user=request.user,
+            reported_by_contributor=contributor,
+            closure_state=request.data.get('closure_state'),
```

Comment:
I was able to submit a report with `closure_state` set to "FOO". As I understand it Django's validation does not kick in unless `full_clean` is called on the model. 

<img width="1018" alt="Screen Shot 2021-02-04 at 10 23 21 AM" src="https://user-images.githubusercontent.com/17363/106930891-4c598580-66d3-11eb-8d0c-cb540a5fa590.png">

Reply:
I added a `full_clean` before `save()` for this call. I ended up adding `blank=True` to a few of the fields that are nullable because they were giving validations errors after adding the check, so I updated the migration.